hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -32,7 +32,7 @@
 import java.util.concurrent.Callable;
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
-
+This
```

Comment:
"This" should be removed. This might be the reason why Yetus is failing.

Reply:
heh, that's what comes of having speech recognition turned on